torch/fx/experimental/subgraph_creation_example.py

```diff
@@ -115,8 +115,8 @@ def record_cross_partition_use(def_node : torch.fx.node.Node, use_node : Optiona
                     if not hasattr(target_attr, atom):
                         raise RuntimeError(f'Operator target {node.target} not found!')
                     target_attr = getattr(target_attr, atom)
-                partition.targets[node.target] = target_attr
                 target = target_atoms[-1]
+                partition.targets[target] = target_attr
```

Comment:
When I think about it more, this might cause collisions (i.e. we're basically stripping off everything in the qualified name except for the last atom, which may be the same across different modules). Does this work?

```
-                partition.targets[node.target] = target_attr
-                target = target_atoms[-1]
+                partition.targets[node.target] = target_attr
```

Reply:
It shows an error on node.py line 31: assert isinstance(target, str)